### PR TITLE
docs: Update sample documentation

### PR DESCRIPTION
**Pull request by DominikPinsel**

### CODE REVIEW

File: docs/samples/Transfer Data.md [hunk 1]
```diff
@@ -3,36 +3,173 @@
...
+> How such an environment can be setup locally is documented in chapter 0.
 
 **Contents**
 
+0. (optional) Local Setup
 1. Setup Data Offer
 2. Request Contract Offers
 3. Negotiate Contract
 4. Transfer Data
 5. Verify Data Transfer
```
Comment (florianrusch-zf) [inline docs/samples/Transfer Data.md:18]: We should also link this entries

File: docs/samples/Transfer Data.md [hunk 2]
```diff
@@ -3,36 +3,173 @@
...
+| cx          | vault                 | No node port    |
+| cx          | vault-internal        | No node port    |
+| cx          | vault-ui              | No node port    |
+| default     | kubernetes            | No node port    |
+| kube-system | kube-dns              | No node port    |
+|-------------|-----------------------|-----------------|---------------------------|
+```
+
+The most important APIs, used by this example, are highlighted. How they are used is described in subchapter 'Set
+Environment Variables, used by this example' below.
```
Comment (florianrusch-zf) [inline docs/samples/Transfer Data.md:66]: The subchapter should also be linked

File: docs/samples/Transfer Data.md [hunk 3]
```diff
@@ -3,36 +3,173 @@
...
+Port:                     metrics  8085/TCP
+TargetPort:               metrics/TCP
+NodePort:                 metrics  31124/TCP
+Endpoints:                172.17.0.6:8085
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:                   <none>
+```
+
+The most important APIs, used by this example, are highlighted. How they are used is described in subchapter 'Set
```
Comment (florianrusch-zf) [inline docs/samples/Transfer Data.md:133]: Again, link the subchapter 😉

File: docs/samples/Local TXDC Setup.md [hunk 4]
```diff
@@ -31,10 +31,10 @@ run connectors, and two different TXDC Connector instances, that can communicate
 
 ### Supporting Infrastructure
 
-Before the connectors can be setup, the Supporting Infrastructure must be in place. The TXDConnector configuration must
-match the different supporting infrastructure components.
+Before the connectors can be setup, the Supporting Infrastructure must be in place. It comes with pre-configured with
+everything to run two connectors independently.
```
Comment (florianrusch-zf) [inline docs/samples/Local TXDC Setup.md:35]:
```suggestion
Before the connectors can be setup, the Supporting Infrastructure must be in place. It comes with pre-configured everything
to run two connectors independently.
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

File: docs/samples/Transfer Data.md [hunk 5]
```diff
@@ -3,36 +3,173 @@
...
+```shell
+|-------------|-----------------------|-----------------|---------------------------|
+|  NAMESPACE  |         NAME          |   TARGET PORT   |            URL            |
+|-------------|-----------------------|-----------------|---------------------------|
+| cx          | backend               | frontend/8080   | http://192.168.49.2:31918 |
+|             |                       | backend/8081    | http://192.168.49.2:30193 | < Transfer Backend API
+| cx          | ids-daps              | No node port    |
+| cx          | plato-controlplane    | default/8080    | http://192.168.49.2:31016 |
+|             |                       | control/8083    | http://192.168.49.2:32510 |
+|             |                       | data/8081       | http://192.168.49.2:30423 | < Plato Data Management API
```
Comment (ndr-brt) [inline docs/samples/Transfer Data.md:45]: the "data" term has been dropped to avoid misunderstandings, now it's called only "management api".

Comment (florianrusch-zf) [reply docs/samples/Transfer Data.md:45]: @ndr-brt @tuncaytunc-zf should we adopt this change maybe in a separate PR for the whole repo?

**Comment by florianrusch-zf** [comment]
As @DominikPinsel is OOO, I will take it over 😉

File: docs/samples/Transfer Data.md (hunk 1 above)
Comment (florianrusch-zf) [inline docs/samples/Transfer Data.md:18]:
```suggestion
0. [(optional) Local Setup](#0--optional--local-setup)
1. [Setup Data Offer](#1-setup-data-offer)
2. [Request Contract Offers](#2-request-contract-offer-catalog)
3. [Negotiate Contract](#3-negotiate-contract)
4. [Transfer Data](#4-transfer-data)
5. [Verify Data Transfer](#5-verify-data-transfer)
```

File: docs/samples/Transfer Data.md (hunk 2 above)
Comment (florianrusch-zf) [reply docs/samples/Transfer Data.md:66]:
```suggestion
The most important APIs, used by this example, are highlighted. How they are used is described in subchapter ['Set
Environment Variables](#set-environment-variables-used-by-this-example), used by this example below.
```

File: docs/samples/Transfer Data.md [hunk 6]
```diff
@@ -3,36 +3,173 @@
...
+TargetPort:               metrics/TCP
+NodePort:                 metrics  31124/TCP
+Endpoints:                172.17.0.6:8085
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:                   <none>
+```
+
+The most important APIs, used by this example, are highlighted. How they are used is described in subchapter 'Set
+Environment Variables, used by this example' below.
```
Comment (florianrusch-zf) [inline docs/samples/Transfer Data.md:134]:
```suggestion
The most important APIs, used by this example, are highlighted. How they are used is described in subchapter ['Set
Environment Variables](#set-environment-variables-used-by-this-example), used by this example below.
```

**Review by florianrusch-zf — APPROVED** [review]

**Pull request merged**